custom_components/envisalink_new/alarm_control_panel.py

```diff
@@ -115,10 +108,9 @@ async def async_setup_entry(
 class EnvisalinkAlarm(EnvisalinkDevice, AlarmControlPanelEntity):
     """Representation of an Envisalink-based alarm panel."""
 
-    _attr_supported_features = (
+    _attr_supported_features: AlarmControlPanelEntityFeature = (
         AlarmControlPanelEntityFeature.ARM_HOME
         | AlarmControlPanelEntityFeature.ARM_AWAY
-        | AlarmControlPanelEntityFeature.ARM_NIGHT
```

Comment:
https://github.com/ufodone/envisalink_new/pull/142#issuecomment-2451241119

Reply:
According to my DSC documentation, `*9` on a keypad does a "zero entry delay".  The description says that it sets the Delay 1 and Delay 2 zones to instant and the Stay/Away zones will remain bypassed.  So this does sound like a Night mode to me.  But it does require a code to be entered. 

I tried this with my own system and it seems to work as expected.  So I think we should leave this mode enabled.

Reply:
About this option, can bring it back as you are right, it's technically possible to night arm a DSC system even tho it does not have a dedicated button on the hw panel.